master/docs/manual/customization.rst

```diff
@@ -932,18 +932,158 @@ For more information on the available commands, see :doc:`../developer/master-wo
     Step Progress
     BuildStepFailed
 
-Writing New Status Plugins
---------------------------
+.. _buildbot_wsgi_dashboards:
+
+Writing Dashboards with Flask_ or Bottle_
+-----------------------------------------
+
+Buildbot Nine UI is written in Javascript.
+This allows it to be reactive and real time, but comes at a price of a fair complexity.
+Sometimes, you need a dashboard displaying your build results in your own manner but learning AngularJS for that is just too much.
+
+There is a Buildbot plugin which allows to write a server side generated dashboard, and integrate it in the UI.
+
+.. code-block:: python
+
+    # This needs buildbot and buildbot_www >= 0.9.5
+    pip install buildbot_wsgi_dashboards flask
+
+- This plugin can use any WSGI compatible web framework, Flask_ is a very common one, Bottle_ is another popular option.
+
+- The application needs to implement a ``/index.html`` route, which will render the html code representing the dashboard.
+
+- The application framework runs in a thread outside of Twisted.
+  No need to worry about Twisted and asynchronous code.
+  You can use python-requests_ or any library from the python ecosystem to access other servers.
```

Comment:
I added an example with requests usage, and a line chart. I hope the example does not get over complicated because of that. please let me know.